templates/index.html

```diff
@@ -1,49 +1,158 @@
 {%include templates/header.html%}
-<div class="hero">
-    <div class="hero-inner">
-        <h1>facebooc</h1>
-        <h2>
-            The best social network you've never heard of!
-        </h2>
-        <div class="btn-group">
-            <a class="btn" href="/signup/"role="button">
-                REGISTER NOW
-            </a>
+<body>
+    <main id="index">
+        <div id="home-page-carousel" class="carousel slide fixed-top" data-bs-ride="carousel">
+            <div class="carousel-indicators">
+                <button type="button" data-bs-target="#home-page-carousel" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
+                <button type="button" data-bs-target="#home-page-carousel" data-bs-slide-to="1" aria-label="Slide 2"></button>
+                <button type="button" data-bs-target="#home-page-carousel" data-bs-slide-to="2" aria-label="Slide 3"></button>
+            </div>
+
+            <div class="carousel-inner">
+                <div class="carousel-item active">
+                    <svg class="bd-placeholder-img" 
+                         width="100%" 
+                         height="100%" 
+                         xmlns="http://www.w3.org/2000/svg" 
+                         aria-hidden="true" preserveAspectRatio="xMidYMid slice" 
+                         focusable="false">
+                        <rect width="100%" height="100%" fill="#3FC1C9"/>
+                    </svg>
+        
+                    <div class="container">
+                        <div class="carousel-caption text-start">
+                            <h1>Facebooc</h1>
+                            <p>The best social network you've never heard of!</p>
+                            <p><a class="btn btn-lg" href="/signup/">Register Now</a></p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="carousel-item">
+                    <svg class="bd-placeholder-img" 
+                         width="100%" 
+                         height="100%" 
+                         xmlns="http://www.w3.org/2000/svg" 
+                         aria-hidden="true" 
+                         preserveAspectRatio="xMidYMid slice" 
+                         focusable="false">
+                        <rect width="100%" height="100%" fill="#3FC1C9"/>
+                    </svg>        
+                    <div class="container">
+                        <div class="carousel-caption">
+                            <h2>What is facebooc?</h2>
+                            <p>A proof-of-concept Facebook clone written in C. The only dependency is SQLite3.</p>
```

Comment:
We might remove "proof-of-concept" because it is not just a proof-of-concept now. 
Also here.